packages/cli/test/unit/ActiveExecutions.test.ts

```diff
@@ -1,24 +1,23 @@
-import * as Db from '@/Db';
 import { ActiveExecutions } from '@/ActiveExecutions';
-import { mocked } from 'jest-mock';
 import PCancelable from 'p-cancelable';
 import { v4 as uuid } from 'uuid';
 import type { IExecuteResponsePromiseData, IRun } from 'n8n-workflow';
 import { createDeferredPromise } from 'n8n-workflow';
 import type { IWorkflowExecutionDataProcess } from '@/Interfaces';
+import { ExecutionRepository } from '@/databases/repositories';
+import Container from 'typedi';
 
 const FAKE_EXECUTION_ID = '15';
 const FAKE_SECOND_EXECUTION_ID = '20';
 
-jest.mock('@/Db', () => {
-	return {
-		collections: {
-			Execution: {
-				save: jest.fn(async () => ({ id: FAKE_EXECUTION_ID })),
-				update: jest.fn(),
-			},
-		},
-	};
+const updateExistingExecution = jest.fn();
+const createNewExecution = jest.fn(async () => {
+	return { id: FAKE_EXECUTION_ID };
+});
+
+Container.set(ExecutionRepository, {
+	updateExistingExecution,
+	createNewExecution,
```

Comment:
For learning, why do we rely on the DI container instead of `mock()` from `jest-mock-extended`?

Reply:
Simply because it seemed easier to do and I followed the example from another file.